main.tf

```diff
@@ -111,7 +111,7 @@ module "additional_vnet_links" {
 }
 
 module "private_endpoint" {
-  source = "git::https://github.com/launchbynttdata/tf-azurerm-module_primitive-private_endpoint.git?ref=1.0.0"
+  source = "git::https://github.com/launchbynttdata/tf-azurerm-module_primitive-private_endpoint.git?ref=feature/output-network-interfaces"
```

Comment:
Reminder to switch this back to a semver tag once the other PR merges, otherwise LGTM.